### ISSUE TITLE
release a 0.6

### ISSUE THREAD

**Issue by christian-rauch** [stevenlovegrove/Pangolin#646]
Since there isn't much development going on on the `master` branch, I would consider the current state more or less stable. Could we maybe release/tag a version `0.6`? I am looking into releasing this as a catkin package and having a fixed tag makes this easier, as other dependant projects could just ask for the specific version 0.6 in their CMakeLists.

**Comment by stevenlovegrove** [comment]
Good idea. I have a refactor at https://github.com/stevenlovegrove/Pangolin/tree/component_refactor which is now fully functional and I have been meaning to merge into master. It has a couple of API breaks in areas I doubt anyone will notice, but that can be a 0.7 or so.

**Comment by stevenlovegrove** [comment]
https://github.com/stevenlovegrove/Pangolin/tree/v0.6
